pkg/api/validation/validation.go

```diff
@@ -1161,7 +1180,7 @@ func ValidateSecret(secret *api.Secret) errs.ValidationErrorList {
 
 func validateBasicResource(quantity resource.Quantity) errs.ValidationErrorList {
 	if quantity.Value() < 0 {
-		return errs.ValidationErrorList{fmt.Errorf("%v is not a valid resource quantity", quantity.Value())}
+		return errs.ValidationErrorList{errs.NewFieldInvalid("value", quantity.Value(), fmt.Sprintf("%v is not a valid resource quantity", quantity.Value()))}
```

Comment:
merge issue?  Rohit should have fixed this earlier today..  @rjnagal did we miss one?